### PR TITLE
Add the ability to background the app

### PR DESCRIPTION
**Pull request by jzucker2**
Uses UIAutomation's ability to background an app for a duration and bubbles that up to the dev for use in tests.

### CODE REVIEW

**Comment by jzucker2** [comment]
Totally open to implementation detail changes, but put this up as a first pass. Overall it's pretty simple so shouldn't require many/if at all changes.

**Comment by phatmann** [comment]
Cool to see the new UIAutomation integration being used for other useful features like this.

Like you, I am concerned about having a new Background controller just for this one test. Can you add the test to System Tests instead, with the notification handler in there?

**Comment by jzucker2** [comment]
Ended up being slightly more complicated to test from that suite, but not too bad. Let me know what you think

**Comment by jzucker2** [comment]
Also the CI is failing but not on my tests

**Comment by phatmann** [comment]
Well, I can see that I steered you wrong on the tests :-( Having a separate view controller really was the cleanest solution. Any chance we can revert back to that?

File: KIF Tests/SystemTests.m [hunk 1]
```diff
@@ -33,6 +33,14 @@ @interface SystemTests : SenTestCase
 
 @implementation SystemTests
 
+- (void)setUp {
+    [[NSNotificationCenter defaultCenter] postNotificationName:@"SystemTestsNotification" object:nil userInfo:@{@"setup" : @YES}];
+}
+
+- (void)tearDown {
+    [[NSNotificationCenter defaultCenter] postNotificationName:@"SystemTestsNotification" object:nil userInfo:@{@"setup" : @NO}];
```
Comment (phatmann) [inline KIF Tests/SystemTests.m]: Did you mean to send "tearDown" here?

Comment (jzucker2) [inline KIF Tests/SystemTests.m]: Ah, that was just being a little lazy when I tried to cram everything into the system tests controller. That code isn't necessary when I revert to the older solution.

**Comment by jzucker2** [comment]
Had some merge conflicts, and decided to move to a different pull request https://github.com/kif-framework/KIF/pull/655

**Pull request closed without merging**